packages/council-tokenlist/.eslintrc

```diff
@@ -6,7 +6,8 @@
   "extends": [
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
-    "prettier"
+    "prettier",
+    "@elementfi/eslint-config"
```

Comment:
Can this file now just be an almost mpty eslint w/ just the global config extension?

FWIW: prettier must be the last extension as per their docs: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

Reply:
Thanks for that tidbit. Good to know for future reference!